### ISSUE TITLE
OS OpenRivers - extends HY-N

### ISSUE THREAD

**Issue by PeterParslow** [INSPIRE-MIF/2017.2#6]
**Simplified encoding example**

# Short description
OS Open Rivers, extends INSPIRE HY-N: names moved to links, a couple of extra attributes

Specification: https://www.ordnancesurvey.co.uk/docs/user-guides/os-open-rivers-user-guide.pdf

Schema: https://www.ordnancesurvey.co.uk/xml/open/rivers/1.0/OSOpenRivers.xsd

The key customers for this were the (England & Wales) Environment Agency and the Scottish Environmetal Protection Agency

Achieved by extending INSPIRE model in EA & then using ShapeChange

# Example instance
Available from https://os.uk/opendatadownload/products.html#OPRVRS

# Purpose & use
tag information with the river IDs used by environment agencies so everybody can use it.

Case study: https://os.uk/business-and-government/case-studies/dundee-city-council-managing-flood-risk.html

# Used simplification rules
Moved names to links

# Additional information

**Comment by michellutz** [comment]
> ## Used simplification rules
> Moved names to links

What do you mean by that exactly?

**Comment by PeterParslow** [comment]
Same as with the premium "OS MasterMap Water Network", we extended the WatercourseLink feature type with a couple of 'name' attributes (although for some reason we gave them different attribute names: localName, localNameAlternative*). We then don't supply any LinkSet features (which is where INSPIRE has the name). We (& our users) feel that this makes the data simpler - at least, simpler to use for the use cases they have in mind. Of course, if they wanted to produce a graphical output, they would have to de-duplicate the names.

The "alternative" name is a different kind of simplification - it restructures the way to highlight a primary name. In many cases, where we have two names, they are used by two different language communities.

We also simplify the names, but not using the INSPIRE GeographicName type, but the GML 3.3 LocalisedCharacterString, which is more widely implemented (being a use of a basic XML approach).

*note: this 'local name' is because it may be a name specific to a short stretch of the watercourse.
